insights/parsers/smbstatus.py

```diff
@@ -0,0 +1,70 @@
+"""
+smbstatus - Command
+===================
+
+This module provides processing for the ``smbstatus`` command. The specs
+handled by this command inlude::
+
+    "smbstatus_S"                 : CommandSpec("/usr/bin/smbstatus -S"),
+    "smbstatus_p"                 : CommandSpec("/usr/bin/smbstatus -p"),
+
+
+Class ``SmbstatusS`` parses the output of the ``smbstatus -S`` command.
+Sample output of this command looks like::
+
+    Service      pid     Machine       Connected at                     Encryption   Signing
+    ----------------------------------------------------------------------------------------
+    share_test   13668   10.66.208.149 Wed Sep 27 10:33:55 AM 2017 CST  -            -
+
+Class ``Smbstatusp`` parses the output of the ``smbstatus -p`` command.
+Sample output of this command looks like::
+
+    Samba version 4.6.2
+    PID     Username     Group        Machine                                   Protocol Version  Encryption           Signing
+    --------------------------------------------------------------------------------------------------------------------------
+    12668   testsmb       testsmb       10.66.208.149 (ipv4:10.66.208.149:44376)  SMB2_02           -                    -
+
+Examples:
+    >>> smbstatusS_info = shared[SmbstatusS]
+    >>> smbstatusS_info.data[0]
+    {'Signing': '-', 'Service': 'share_test1', 'Encryption': '-', 'pid': '12668', 'Machine': '10.66.208.149', 'Connected': 'Wed Sep 27', 'at': '10:33:55 AM 2017 CST'}
+    >>> smbstatusS_info.data[0]['PID']
+    '13668'
+"""
+
+from .. import parser, Parser, get_active_lines
+from . import parse_fixed_table
+
+
+@parser('smbstatus_S')
+class SmbstatusS(Parser):
+    """
+        Class for ``smbstatus -S`` command.
+
+        The format of ``smbstatus -S`` is like table, and function `parse_fixed_table`
+        could parse it.
+
+        Attributes:
+            data (list): List of dicts, where the keys in each dict are the column
+            headers and each item in the list represents a connection.
+    """
+
+    def parse_content(self, content):
+        self.data = parse_fixed_table(get_active_lines(content), header_substitute=[('Connected at', 'Connected_at')])[1:]
```

Comment:
Better use get_active_lines(content, '----') to skip the '-----------' lines, instead of using `[1:]`